satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -361,6 +361,67 @@ read all (
     orderby asc api_key.name
 )
 
+//-----local_payments-----//
+
+model local_invoice (
+    key id
+
+    field id  blob
+
+    field payment_method_id  blob
+    field amount             int64
+    field currency           text
+    field status             text
+
+    field  created_at  timestamp  (autoinsert)
+)
+
+create local_invoice()
+
+read one (
+    select local_invoice
+    where local_invoice.id = ?
+)
+
+model local_invoice_line_item (
```

Comment:
Maybe if the line items were serialized via protobuf or something, life would be much easier.